### ISSUE TITLE
Try clustering on annotators to partition t

### ISSUE THREAD

**Issue by MatthewJA** [chengsoonong/crowdastro#138]
#82

**Comment by MatthewJA** [comment]
Problem with clustering annotators: Labels are only partially observed and mostly don't overlap. Will try anyway.

**Comment by MatthewJA** [comment]
Balanced accuracy (with `sklearn.linear_model.LogisticRegression`, testing on Norris) for each of five clusters:

```
0.727235774647
0.715165678675
0.754294782614
0.723188656146
0.5
```

**Comment by MatthewJA** [comment]
Balanced accuracy with Yan et al. is 0.497 with confusion matrix

```
[[   0 1589]
 [   1  153]]
```

:/

**Comment by MatthewJA** [comment]
Weights for each cluster:

![image](https://cloud.githubusercontent.com/assets/2681312/17654737/a9bd9934-62e9-11e6-9d14-207e6a2a2ff7.png)

γ for each cluster:

`[ -206.06684877  -995.36668545 -1651.16336611 -1076.5866288    -48.6273634 ]`

I think that reflects an expectation that the labellers will choose 0. I'm not sure why.

**Comment by MatthewJA** [comment]
Anyway, done with this issue for now.
